waku-bindings/src/node/lightpush.rs

```diff
@@ -0,0 +1,49 @@
+//! Waku lightpush protocol related methods
+
+// std
+use std::ffi::CString;
+// crates
+use libc::*;
+// internal
+use crate::general::{MessageHash, Result, WakuMessage};
+use crate::node::events::WakuNodeContext;
+use crate::utils::{get_trampoline, handle_response, LibwakuResponse};
+
+pub fn waku_lightpush_publish_message(
+    ctx: &WakuNodeContext,
+    message: &WakuMessage,
+    pubsub_topic: &str,
```

Comment:
Even if we use string for topics in nwaku I would discourage this use in rust.

string bad mkay?

Reply:
> Even if we use string for topics in nwaku I would discourage this use in rust.
> 
> string bad mkay?

Sorry but I can't quite follow the suggestion :)

Reply:
Create a type `PubsubTopic` and use it everywhere basically.